bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -166,6 +166,11 @@ trait ItcTable extends Table {
 
   }
 
+  // lacking a simple way to decide on the "closest" (i.e. "fastest to reach") peer we talk to the observing peer
+  // if defined and GN or GS otherwise, if no site is defined you're out of luck and the ITC tables will stay empty
+  protected def itcServicePeer: Option[Peer] =
+    ObservingPeer.get orElse ObservingPeer.peerFor(Site.GN) orElse ObservingPeer.peerFor(Site.GS) orElse None
```

Comment:
Is this the server to talk to? If so shouldn't we fallback to something? I presume most users are on the northern hemisphere so defaulting to GN would make sense